apps/src/lib/levelbuilder/script-editor/ScriptEditor.jsx

```diff
@@ -234,7 +232,7 @@ class ScriptEditor extends React.Component {
       project_widget_visible: this.state.projectWidgetVisible,
       project_widget_types: this.state.projectWidgetTypes,
       lesson_extras_available: this.state.lessonExtrasAvailable,
-      script_text: this.props.beta
+      script_text: this.props.isMigrated
```

Comment:
Does `migrated` need to be in `dataToSave` in order to not lose it?

Reply:
Yup. Its the last thing in dataToSave.